src/ShadowDOM/wrappers/Range.js

```diff
@@ -20,23 +20,69 @@
 
   var OriginalRange = window.Range;
 
+  var ShadowRoot = scope.wrappers.ShadowRoot;
+
+  function isElement(node){
```

Comment:
Do you have a tool to check the style? I am using Intellij and I am trying to set the style rules to detect webcomponents code style, but it seems I am still off.